<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم  الاول</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الهـــــاتف</string>
   </property>
  </widget>
  <widget class="QPushButton" name="labAddFathers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(4, 0, 62);
color:rgb(255, 255, 255);
font: 15pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>إضافة عضو جديد</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateCouncilDOB">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="locale">
    <locale language="Arabic" country="Yemen"/>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>تاريخ الميلاد</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilFatherfName">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>100</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 9pt &quot;Motken K Sina&quot;;
    color:rgb(79, 178, 239);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilPhone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 9pt &quot;Motken K Sina&quot;;
    color:rgb(79, 178, 239);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>153</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>اللقب</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilLName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>153</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 9pt &quot;Motken K Sina&quot;;
    color:rgb(79, 178, 239);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilOrganicStatus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>444</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 9pt &quot;Motken K Sina&quot;;
    color:rgb(79, 178, 239);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>347</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الحالة الاجتماعية</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilAddrress">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>394</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 9pt &quot;Motken K Sina&quot;;
    color:rgb(79, 178, 239);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilSocialStatus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>344</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 9pt &quot;Motken K Sina&quot;;
    color:rgb(79, 178, 239);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>444</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>المكانة العضوية</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>397</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>العنوان</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combFatherGender">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>291</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
 color:rgb(79, 178, 239);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	
	border: 1px solid #5b586e;

</string>
   </property>
   <item>
    <property name="text">
     <string>ذكر</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>أنثى</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>205</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الجنس</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveCouncil">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>103</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>حفــظ</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/true.png</normaloff>:/icons_qrc/true.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingCouncil">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>103</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Motken K Sina</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>الغــاء</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/exclamation-48.ico</normaloff>:/icons_qrc/exclamation-48.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCloseDialog">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0.506975, y1:0.432, x2:0.99, y2:0.437227, stop:0.900498 rgba(4, 0, 62, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/multiply.png</normaloff>:/icons_qrc/multiply.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtCouncilFatherfName</tabstop>
  <tabstop>txtCouncilLName</tabstop>
  <tabstop>txtCouncilPhone</tabstop>
  <tabstop>dateCouncilDOB</tabstop>
  <tabstop>labAddFathers</tabstop>
 </tabstops>
 <resources>
  <include location="icons_rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
